packages/react-native/src/transport.ts

```diff
@@ -33,7 +33,8 @@ export class Transport implements Types.Transport {
   }
 
   private buildUserAgent() {
-    const reactNativeVersion = `${Platform.constants.reactNativeVersion.major}.${Platform.constants.reactNativeVersion.minor}.${Platform.constants.reactNativeVersion.patch}`
+    const rnVersion = Platform.constants?.reactNativeVersion
+    const rnVersionStr = `${rnVersion?.major}.${rnVersion?.minor}.${rnVersion?.patch}`
     
     let nativePlatform:string
```

Comment:
```suggestion
    let nativePlatform: string
```
🤔 Shouldn't this get picked up from eslint?

Reply:
I confirmed that this file is getting linted. 

Looks like [type-annotation-spacing](https://typescript-eslint.io/rules/type-annotation-spacing/) is not part of `@typescript-eslint/recommended`. They want you to use Prettier or another formatter instead. I'm also partial to using Prettier, but that is too big a task for this PR, so I'll just go ahead and enable this rule for now.